evo_researcher/functions/prepare_report.py

```diff
@@ -14,6 +16,10 @@ def prepare_summary(goal: str, content: str, model: str):
 
 {content}
 """
+    encoder = tiktoken.encoding_for_model(model)
+    while trim_content_to_tokens and len(encoder.encode(content)) > trim_content_to_tokens:
```

Comment:
Could do in one go with encode -> trim -> decode ?

Reply:
Sure, changed